core/transfer/src/test/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImplIntegrationTest.java

```diff
@@ -62,6 +63,7 @@ void setup() {
                 .dispatcherRegistry(mock(RemoteMessageDispatcherRegistry.class))
                 .manifestGenerator(manifestGenerator)
                 .monitor(mock(Monitor.class))
+                .telemetry(mock(Telemetry.class))
```

Comment:
nit: strange to mock such a class, isn't it better to use the default no-op telemetry?